internal/output/markdown.go

```diff
@@ -227,7 +235,7 @@ func ToMarkdown(out Root, opts Options, markdownOpts MarkdownOptions) ([]byte, e
 	})
 	_, err := tmpl.ParseFS(templatesFS, "templates/"+filename)
 	if err != nil {
-		return []byte{}, err
+		return MarkdownOutput{Msg: []byte{}, RuneLen: 0, OriginalMsgSize: 0}, err
```

Comment:
FYI these are the same as having the zero values, it is essentially equal to `MarkdownOutput{}`

Reply:
Oh right! 🤦 Sorry for all the noob issues and thanks for the patience 🙏